certified-connectors/intelliHR/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7967 @@
+            },
+            "examples": {
+              "application/json": {
+                "value": {
+                  "name": "Forbidden",
+                  "message": "Error",
+                  "code": 403,
+                  "errors": {}
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {
+              "$ref": "#/definitions/InternalServerError"
+            },
+            "examples": {
+              "application/json": {
+                "value": {
+                  "name": "Internal Server Error",
+                  "message": "An internal server error occurred and has been recorded. Please try again later. If this does not resolve your issue, please contact support@intellihr.co for support and guidance.",
+                  "code": 500,
+                  "errors": {}
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/custom-field-definitions": {
+      "get": {
+        "description": "Returns a list of all Custom Field Definitions.",
+        "summary": "List all Custom Field Definitions",
+        "tags": [
+          "Custom Field Definitions"
+        ],
+        "operationId": "ListallCustomFieldDefinitions",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "limit",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "minimum": 1,
+            "maximum": 100,
+            "exclusiveMaximum": false,
+            "exclusiveMinimum": false,
+            "description": "The number of items per page",
+            "x-ms-summary": "The number of items per page"
+          },
+          {
+            "name": "page",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "minimum": 1,
+            "exclusiveMinimum": false,
+            "description": "The page number",
+            "x-ms-summary": "The page number"
+          },
+          {
+            "name": "filters",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Filters to apply to the query",
+            "description": "Filters can be applied in the query string to limit the data returned from this endpoint. Filters are provided in the format `filters[<filter_name>][<operation_type>]=<filter_value>`. For example, a filter to get items matching a specific ID would be `filters[id][eq]=8a5f3ea6-ea6b-4425-8a87-3c256bb7b6f9`\n\nBelow is a list of operation types available on filters, note that each filter may not support every type:\n\n\u2022 Equality `[eq]`: Checks for an exact match with the given filter value\n\nThe list of supported filters is given below."
```

Comment:
@wolskis This description has unicodes in the value. Can you please remove all the unicodes from the swagger as unicodes may not supported in all the Power Automate pages

Reply:
Unicodes removed